api/src/middleware/__tests__/competenciesRouter.ts

```diff
@@ -11,6 +13,8 @@ jest.mock('../../services/competenciesService.ts');
 const mockCountCompetencies = jest.mocked(countCompetencies);
 const mockListCompetencies = jest.mocked(listCompetencies);
 const mockCreateCompetency = jest.mocked(createCompetency);
+const mockUpdateCompetency = jest.mocked(updateCompetency);
+const mockAuthorizeCompetencyUpdate = jest.mocked(authorizeCompetencyUpdate);
```

Comment:
Should these variables also be listed alphabetically like their imports?

Reply:
Amended to alphabetical order as suggested.